changelogs/unreleased/gh-6766-fix-error-in-extension-via-netbox.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/sql
```

Comment:
In the commit message which I see in git (github displays it differently somewhy) you say `Closes tarantool#6766`. Lets better use the more canonic way `Closes #6766`.

Reply:
Thanks! Fixed.